qemu/tests/cfg/blockdev_commit_install.cfg

```diff
@@ -0,0 +1,40 @@
+- blockdev_commit_install:
+    type = blockdev_commit_install
+    virt_test_type = qemu
+    only Linux
```

Comment:
We may need to make it clear that this case is designed for rhel8.2+

Reply:
have updated.